src/icons/SealLogo.tsx

```diff
@@ -0,0 +1,59 @@
+import classnames, {
+  borderRadius,
+  boxShadow,
+  boxShadowColor,
+  fill,
+  height,
+  stroke,
+  strokeWidth,
+  width,
+} from 'classnames/tailwind'
+
+const svgClasses = classnames(
+  boxShadow('shadow-lg'),
+  boxShadowColor('shadow-secondary'),
+  borderRadius('rounded-full'),
+  width('w-16'),
```

Comment:
Maybe we should specify the size outsize the svg? So we can re-use the icon anytime without modifying it?

Reply:
I think, there is no need for this right now. We use it in one place